picture_in_picture/app/build.gradle.kts

```diff
@@ -0,0 +1,84 @@
+plugins {
+    id("com.android.application")
+    id("org.jetbrains.kotlin.android")
+    id("org.jlleitschuh.gradle.ktlint")
+    id("io.gitlab.arturbosch.detekt")
+}
+
+android {
+    namespace = "com.firework.example.pictureinpicture"
+
+    compileSdk = 33
+
+    defaultConfig {
+        applicationId = "com.firework.example.pictureinpicture"
+
+        minSdk = 21
+        targetSdk = 33
+
+        versionCode = 1
+        versionName = "1.0.0"
+    }
+
+    buildTypes {
+        defaultConfig {
+            buildConfigField("String", "FW_CLIENT_ID", "\"f6d6ec1275217f178cdff91363825cb390e038c1168f64f6efa23cb95ec6b325\"")
+            buildConfigField("String", "FW_CHANNEL_ID", "\"7RXwK8k\"")
+            buildConfigField("String", "FW_PLAYLIST_ID", "\"g4lA0g\"")
```

Comment:
These values are not beign used in the sample app
```suggestion
```

Reply:
Copy/Paster effect, removed